samples/BenchmarkDotNet.Samples.Forms/MainPage.xaml.cs

```diff
@@ -0,0 +1,61 @@
+﻿using BenchmarkDotNet.Analysers;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Running;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace BenchmarkDotNet.Samples.Forms
+{
+	public partial class MainPage : ContentPage
+	{
+		public MainPage()
+		{
+			InitializeComponent();
+		}
+
+		async void Button_Clicked(object sender, EventArgs e)
+		{
+			SetIsRunning(true);
+			try
+			{
+				var logger = new AccumulationLogger();
+				await Task.Run(() =>
+				{
+					var summary = BenchmarkRunner.Run<IntroBasic>();
+					MarkdownExporter.Console.ExportToLog(summary, logger);
+					foreach (var benchmark in summary.BenchmarksCases)
+					{
+						ConclusionHelper.Print(logger, benchmark.Config.GetCompositeAnalyser().Analyse(summary).Distinct().ToList());
+					}
```

Comment:
in most of the cases, all benchmark instances are going to share the same instance of `Config`. To avoid duplicates we should aggregate the conclusions for all of them and then call distinct

```suggestion
        ConclusionHelper.Print(logger, 
            summary.BenchmarksCases
                   .SelectMany(benchmark => benchmark.Config.GetCompositeAnalyser().Analyse(summary))
                   .Distinct()
                   .ToList()
       );
```